.github/workflows/docs.yml

```diff
@@ -78,8 +78,8 @@ jobs:
         uses: mxschmitt/action-tmate@v3
 
       - name: Deploy 🚀
-        uses: JamesIves/github-pages-deploy-action@4.1.5
-        if: steps.documentation.outputs.status == 'done' && github.ref == 'refs/heads/master'
+        uses: JamesIves/github-pages-deploy-action@v4
+        if: env.OK_TO_DEPLOY_DOCS == 'true' && github.ref == 'refs/heads/master'
```

Comment:
All the other references to environment variables in these files use the `${{ env.OK_TO_DEPLOY_DOCS }}` syntax - I'm not sure it's required, but it might be nice to stick to one consistent syntax.